test/e2e/testdata/sensor-atmostonce-failing.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Sensor
+metadata:
+  name: e2e-atmostonce-failing
```

Comment:
I think the sensor name needs to be the same for both
- e2e-atleastonce-triggerable
- e2e-atleastonce-failing

Otherwise a completely new subscription to jetstream will be created and the ack (which should have occurred in the at most once case) will be ignored.